docs/general/installation/synology.md

```diff
@@ -5,7 +5,67 @@ description: Install on Synology NAS.
 sidebar_position: 4
 ---
 
-For [Synology](https://www.synology.com/en-us/dsm), Jellyfin is installed using Docker.
+# Install on Synology
+
```

Comment:
I navigate towards including a disclaimer pointing towards our hardware selection guide with a clear warning that Prebuild nas systems are generally discouraged to be used.

```suggestion
:::caution Pre-built NAS Devices

Many pre-built NAS devices are underpowered. We generally do not recommend running Jellyfin on those devices.
See: [Hardware Selection](https://jellyfin.org/docs/general/administration/hardware-selection#normal-server) for more information.

:::
```